### PR TITLE
Preparing for PyPI packaging

### PR DESCRIPTION
**Pull request by spokerman12**
# Preparing  Flow Control XBlock for PyPI packaging

## Background:

PyPI asks for its contributors' projects to meet certain [distribution guidelines](https://circleci.com/blog/Adding-container-security-scanning-anchore/). We implemented these changes accordingly.

## Context:

- We fixed up Flow Control to meet PyPI's requirements. A preview is available [here](https://test.pypi.org/project/flow-control-xblock/0.2.0/).
- Added setup.py package data
- Version is obtained from ``flow_control/__init__.py``
- Changed name from xblock-flow-control to flow-control-xblock, to match edX's practices.
- 

## Additional changes

- Updated the badge to CircleCI rather than Travis
- Fixed README.rst formatting to fit setup.py's long_description.

## Reviewers
@morenol 

## Next steps
- Enabling automatic version checks
- Upgrade to Python 3

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.PHONY: help test install upgrade
+.PHONY: help test install upgrade package requirements
```
Comment (morenol) [inline Makefile:2]: Let's do this changes(if needed) in the PR with the circleci-pypi integration

File: setup.py [hunk 2]
```diff
@@ -69,15 +79,43 @@ def load_requirements(*requirements_paths):
...
+    long_description=long_description,
+    long_description_content_type='text/x-rst',
+    url='https://github.com/eduNEXT/flow-control-xblock',
+    packages=find_packages(),
+    classifiers=[
+        'Intended Audience :: Developers',
+        'Intended Audience :: Education',
+        'License :: OSI Approved :: GNU Affero General Public License v3',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
```
Comment (morenol) [inline setup.py:104]: Add the python2 and python 2.7 and Django1.11 classifiers

File: Makefile (hunk 1 above)
Comment (spokerman12) [reply Makefile:2]: Noted. Removed.

File: setup.py [hunk 3]
```diff
@@ -86,7 +124,11 @@ def load_requirements(*requirements_paths):
             'flow-control = flow_control:FlowCheckPointXblock',
         ],
     },
+    keywords='edx edunext xblock flowcontrol flow-control',
```
Comment (morenol) [inline setup.py]: Let's remove the edx keyword in order to avoid trademark issues https://open.edx.org/trademark-licensing-details/

File: setup.py [hunk 4]
```diff
@@ -69,15 +79,43 @@ def load_requirements(*requirements_paths):
         )
     return list(requirements)
 
+
+class VerifyVersionCommand(install):
```
Comment (morenol) [inline setup.py]: Let's do this changes(if needed) in the PR with the circleci-pypi integration

Comment (spokerman12) [reply setup.py]: Noted. Will remove these for now

File: setup.py (hunk 3 above)
Comment (spokerman12) [reply setup.py]: Removed.

File: setup.py (hunk 2 above)
Comment (spokerman12) [reply setup.py:104]: Added.

Comment (morenol) [reply setup.py:104]: Add also the python 2.7 classifier

**Review by morenol — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (spokerman12) [reply setup.py:104]: Added! Missed it during git add

File: flow_control/__init__.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
 """
-This is the FlowControl Xblock package
+Init for main Flow-Control XBlock
 """
-from .flow import FlowCheckPointXblock
```
Comment (morenol) [inline flow_control/__init__.py:4]: Why did you remove this import?

Comment (spokerman12) [reply flow_control/__init__.py:4]: My bad! Added back.
